.travis.yml

```diff
@@ -10,9 +10,11 @@ addons:
     - libssl-dev
     - libunwind8
     - zlib1g
-env:
-  - KOREBUILD_DNU_RESTORE_CORECLR=true KOREBUILD_TEST_DNXCORE=true
 mono:
   - 4.0.5
+os:
+  - linux
+  - osx
```

Comment:
Should include install step for OSx that does `brew install icu4c`. See aspnet/Universe/issues/349.